### PR TITLE
SDIT-2089 NOMS API DTO regeneration

### PR DESCRIPTION
**Pull request by andymarke**
Required due to name clashes

### CODE REVIEW

File: src/test/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/contactperson/ContactPersonNomisApiMockServer.kt [hunk 1]
```diff
@@ -90,11 +102,30 @@ fun contactPerson(): ContactPerson = ContactPerson(
...
     createUsername = "Q1251T",
   ),
-  phoneNumbers = emptyList(),
-  addresses = emptyList(),
-  emailAddresses = emptyList(),
-  employments = emptyList(),
-  identifiers = emptyList(),
-  contacts = emptyList(),
-  restrictions = emptyList(),
+  phoneNumbers = listOf(PersonPhoneNumber(phoneId = 1, number = "0114555555", type = CodeDescription(code = "HOME", description = "Home"))),
```
Comment (andymarke) [inline src/test/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/contactperson/ContactPersonNomisApiMockServer.kt:105]: This effectively tests we are using the correct DTOs

**Review by petergphillips — APPROVED** [review]
LGTM

File: src/test/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/activities/AllocationMigrationIntTest.kt [hunk 2]
```diff
@@ -517,9 +517,9 @@ class AllocationMigrationIntTest : SqsIntegrationTestBase() {
     @Test
     fun `will cancel a running migration`() {
       // slow the API calls so there is time to cancel before it completes
-      nomisApi.setGlobalFixedDelay(1000)
-      stubMigrationDependencies(entities = 10)
-      mappingApi.stubAllocationsMappingByMigrationId(count = 10)
+      nomisApi.setGlobalFixedDelay(20)
```
Comment (andymarke) [inline src/test/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/activities/AllocationMigrationIntTest.kt:520]: another attempt to fix test completing before it is cancelled

**Review by petergphillips — APPROVED** [review]
LGTM

**Pull request merged**